src/ray/pubsub/publisher.h

```diff
@@ -55,16 +59,13 @@ class SubscriptionIndex {
   /// Return true if the object id exists in the index.
   bool HasObjectId(const ObjectID &object_id) const;
 
+  /// Returns true if object id or subscriber id exists in the index.
+  bool HasSubscriber(const SubscriberID &subscriber_id) const;
```

Comment:
Moved to the public because FRIEND_TEST doesn't seem to work well with different namespace. I think it makes sense that it is the public interface (`HasObjectId` is also a public interface now because it is used).